packages/libs/deer/macros/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "deer-macro"
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
Shall we delete this too? No need to leave it in

Reply:
removed in [14cde13](https://github.com/hashintel/hash/pull/1163/commits/14cde13667758ee6058e6bdcec299c5e8ba71c40)